spring-cloud-stream/pom.xml

```diff
@@ -35,6 +35,10 @@
 			<artifactId>spring-integration-core</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-jmx</artifactId>
```

Comment:
On the long run, I'd prefer if the Boot feature would not require JMX, but that's something we need to address in Boot.